iati/core/default.py

```diff
@@ -133,18 +134,42 @@ def codelists(version=None):
     """Locate the default Codelists for the specified version of the Standard.
 
     Args:
-        version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that the latest version of the Codelist is returned.
-
-    Raises:
-        ValueError: When a specified version is not a valid version of the IATI Standard.
+    version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that the latest version of the Codelist is returned.
 
     Returns:
-        dict: A dictionary containing all the Codelists at the specified version of the Standard. All Non-Embedded Codelists are included. Keys are Codelist names. Values are iati.core.Codelist() instances.
+    dict: A dictionary containing all the Codelists at the specified version of the Standard. All Non-Embedded Codelists are included. Keys are Codelist names. Values are iati.core.Codelist() instances.
 
     """
     return _codelists(version)
 
 
+def ruleset(version=None):
```

Comment:
The Standard Ruleset needs added to populated Schemas in `_populate_schema()` (and checks adding to the relevant test(s) that populated Schemas include a Ruleset).

Reply:
done